lib/widgets/banner/banner.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+enum FlexBannerImageLayout {
+  /// Image is firstly visible, the rest of the element are shown after the
+  /// image (i.e. on the right if the orientation is horizontal, at the bottom
+  /// if orientation is vertical)
+  first,
+
+  /// Image is lastly visible, the rest of the element are shown before the
+  /// image (i.e. on the left if the orientation is horizontal, at the top
+  /// if orientation is vertical)
+  last,
+}
+
+class FlexBanner extends StatelessWidget {
+  const FlexBanner({
+    super.key,
+    required this.title,
+    required this.image,
+    this.orientation = Axis.horizontal,
+    this.imageLayout = FlexBannerImageLayout.last,
+    this.theme,
+    this.heading,
+    this.description,
+    this.button,
+  });
+
+  final String title;
+
+  final FlexImage image;
+
+  final Axis orientation;
+
+  final FlexBannerImageLayout imageLayout;
+
+  final CardTheme? theme;
+
+  // TODO Implement this, I'm thinking a heading line above the title to emphasise the title
+  final String? heading;
+
+  // TODO Implement this
+  final String? description;
+
+  /// TODO Make a standard button. Also: should be a list of buttons for more flexibility
```

Comment:
Yes, should definitely be standard instead of AddToCart, unless there are specific properties (in which case we can simply extend)